router-tests/cmd/config-validator/extract_schemas.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"slices"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Configuration struct {
```

Comment:
Please use the auto generated config types.

Reply:
Discussed and addressed,